src/main/java/com/google/sps/data/Listing.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.data;
+
+/** Class for Listing object with the specified attributes returned from the places API **/
+public class Listing{
```

Comment:
Super, mega, big, important nit .... space before the bracket :)

Reply:
Ugh! How could I miss that!!